File: kata-5-kyu/simple-pig-latin/javascript/simplePigLatin.js
//+ ====================================================================================================================
//+
//+ 5 kyu - Simple Pig Latin  [ ID: 520b9d2ad5c005041100000f ] (simple-pig-latin)
//+ URL: https://www.codewars.com/kata/520b9d2ad5c005041100000f
//+ Category: ALGORITHMS  |  Tags: REGULAR EXPRESSIONS | ALGORITHMS
//+
//+ ====================================================================================================================

function pigIt(str) {
  return str.replace(/\b(\w)(\w*)\b/g, "$2$1ay")
}

module.exports = { pigIt }

File: kata-5-kyu/simple-pig-latin/python/simple_pig_latin.py
# + =====================================================================================================================
# +
# + 5 kyu - Simple Pig Latin  [ ID: 520b9d2ad5c005041100000f ] (simple-pig-latin)
# + URL: https://www.codewars.com/kata/520b9d2ad5c005041100000f
# + Category: ALGORITHMS  |  Tags: REGULAR EXPRESSIONS | ALGORITHMS
# +
# + =====================================================================================================================

from re import sub


def pig_it(text):
    return sub(r"\b(\w)(\w*)\b", r"\2\1ay", text)
